src/Template/Template.php

```diff
@@ -189,7 +189,13 @@ public function start($name)
             );
         }
 
-        $this->sections[$name] = '';
+        if (isset($this->sections[$name])) {
+            $content = $this->sections[$name];
+            unset($this->sections[$name]);
+            $this->sections[$name] = $content;
```

Comment:
You got it. And yes, just using a template parameter is a better solution. I'll update the PR.